src/aap_eda/conf/registry.py

```diff
@@ -242,14 +242,22 @@ def _resync_remote_settings(self, sync_group: str) -> None:
         url = f"{settings.RESOURCE_SERVER['URL']}/{sync_slug}"
         token = resource_server.get_service_token()
         logger.info(f"Getting remote settings from {url}")
-        res = requests.get(
-            url,
-            headers={"X-ANSIBLE-SERVICE-AUTH": token},
-            verify=settings.RESOURCE_SERVER["VALIDATE_HTTPS"],
-        )
+        try:
+            res = requests.get(
+                url,
+                headers={"X-ANSIBLE-SERVICE-AUTH": token},
+                verify=settings.RESOURCE_SERVER["VALIDATE_HTTPS"],
+            )
+        except Exception as e:
```

Comment:
Is there a reason to catch a generic exception?

Reply:
to catch any exception from requests I think you can just catch `RequestException` which is parent of the rest. 
https://github.com/psf/requests/blob/main/src/requests/exceptions.py#L12